mscore/file.cpp

```diff
@@ -2220,11 +2221,17 @@ Score::FileError readScore(MasterScore* score, QString name, bool ignoreVersionE
                   qDebug("unknown file suffix <%s>, name <%s>", qPrintable(suffix), qPrintable(name));
                   return Score::FileError::FILE_UNKNOWN_TYPE;
                   }
+            if (score->isInvalid())
```

Comment:
Why does this need to be checked here? Isn't once at the end of this clause sufficient (i.e. just the once at line 2232)?

Reply:
Well, initially I placed it here to prevent calling `connectTies` on invalid score but looking at `connectTies` now I suppose it is safe to call this member function on bad scores. So the second check should probably be sufficient. I'll update this pull request, thank you!